glass-progress-bar/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:0.9.+'
```

Comment:
@barbeau Should we use gradle version ```1.0.1``` as same with OBA (https://github.com/CUTR-at-USF/onebusaway-android/blob/glass/onebusaway-android/build.gradle)?